core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -168,9 +168,9 @@ public synchronized void connect() throws AlluxioStatusException {
     disconnect();
     Preconditions.checkState(!mClosed, "Client is closed, will not try to connect.");
 
-    RetryPolicy retryPolicy =
-        ExponentialTimeBoundedRetry.builder().withMaxDuration(MAX_RETRY_DURATION)
-            .withInitialSleep(BASE_SLEEP_MS).withMaxSleep(MAX_SLEEP_MS).build();
+    RetryPolicy retryPolicy = ExponentialTimeBoundedRetry.builder()
+            .withMaxDuration(MAX_RETRY_DURATION).withInitialSleep(BASE_SLEEP_MS)
```

Comment:
indentation is off

Reply:
Yes, seems that by mistake I modified the original indentation while merging with the master branch, I will fix it, thanks!